apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -2,6 +2,9 @@
  * Generated by `createschema news.OrganizationNewsItem 'organization:Relationship:Organization:CASCADE; title:Text; body:Text; type:Select:common,emergency'`
  */
 
+const BadWordsNext = require('bad-words-next')
+const badWordsRu = require('bad-words-next/data/ru.json')
```

Comment:
where's `en`?

Reply:
Don't think this is necessary, but done.

Reply:
Hm, no, just removed english. Faker sometimes generates bad words
<img width="1421" alt="image" src="https://user-images.githubusercontent.com/25384290/235128438-24290d41-eb79-4a14-b9cb-19de45cf16e8.png">